import java.util.Scanner;
public class Main
{
	public static void main(String[] args) {
		
		System.out.println("start");
		System.out.println("Progress");
        int a=100;
        try{
            System.out.println(a/0);//ArithmeticException
        }
        catch(Exception e){
            System.out.println("ArithmeticException occoured and handled sucessfully");
        }
        finally
        {
            System.out.println("here is a finally block");
        }
		System.out.println("completed");
		System.out.println("exited");
		
	}
}
